app/stac_api/management/commands/dummy_asset_upload.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import AssetUpload
+from stac_api.models import BaseAssetUpload
+from stac_api.s3_multipart_upload import MultipartUpload
+from stac_api.utils import AVAILABLE_S3_BUCKETS
+from stac_api.utils import CommandHandler
+from stac_api.utils import get_sha256_multihash
+
+logger = logging.getLogger(__name__)
+
+
+class DummyAssetUploadHandler(CommandHandler):
+
+    def __init__(self, *args, **kwargs):
+
+        s3_bucket = kwargs.pop('s3_bucket', AVAILABLE_S3_BUCKETS.legacy)
```

Comment:
So this seems as a preparation for chosing another bucket than legacy? I don't quite follow where this `kwargs` would be set, as there is no argument parse option